.gitignore

```diff
@@ -80,3 +80,5 @@ yarn-debug.log*
 yarn-error.log*
 .vscode/launch.json
 package-lock.json
+
+.idea/
```

Comment:
sure it can't hurt!

what is this for, out of curiosity :)

Reply:
If you are using IDE from JetBrains (I use WebStorm for web stack) - the IDE put own files into `.idea` path. It is unnecessary put those files into repository. This row hide path and subfiles for GIT version control system